public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -229,14 +229,10 @@ export function getServiceMapGraph(
         borderWidth: 3,
         color: {
           border: '#4A4A4A',
-          background:
-            relatedServices!.indexOf(service) >= 0 ? `rgba(${color}, 1)` : `rgba(${color}, 0.2)`,
```

Comment:
since it's not using `relatedServices` anymore, remove the `relatedServices` in the `getServiceMapGraph()` parameter and where it's passed in to `getServiceMapGraph()`?

Reply:
Removed

Reply:
Removed